ports/raspberrypi/common-hal/microcontroller/Pin.h

```diff
@@ -34,10 +34,13 @@
 
 #include "peripherals/pins.h"
 
+extern bool cyw_ever_init;
+
 void reset_all_pins(void);
 // reset_pin_number takes the pin number instead of the pointer so that objects don't
 // need to store a full pointer.
 void reset_pin_number(uint8_t pin_number);
+void reset_pin_number_cyw(uint8_t pin_number);
```

Comment:
Guard with `CIRCUITPY_CYW43` so it's completely not there if compiling for regular Pi Pico?